RELEASE_NOTES.md

```diff
@@ -0,0 +1,70 @@
+* Improved binary post-op performance with `per_w` broadcast.
+
+## Intel Graphics Products
+* Improved performance for Intel GPUs based on Xe3 architecture.
+* Improved convolution performance on Intel Arc Graphics for Intel Core Ultra processors (Series 2) (formerly Lunar Lake) and Intel Arc B-series discrete graphics (formerly Battlemage).
+* Improved `int8` matmul performance with zero points for source and weight tensors.
+* Improved performance of the following subgraphs with Graph API:
+	* Scaled Dot Product Attention (SDPA) with `int4` and `int8` KV cache.
+	* SDPA with bottom-right implicit causal mask.
+	* SDPA with head size between 257 and 512.
+	* Grouped Query Attention (GQA) with 5D input tensors.
+
+## AArch64-based Processors
+* Enabled BF16 forward-mode inner product via ACL. 170x and 160x speedups for BERT and AlexNet in torch compile-mode.
+* Enabled post-ops in jit_sve_1x1 conv. Yields 1.2x-1.4x speedups for 1x1 kernel problems.
+* Prefer jit_sve conv where faster. Yields 1.3x-3x speedups on average.
+
+# Functionality
+## Common
+* Introduced select algorithm support in [binary primitive](https://uxlfoundation.github.io/oneDNN/v3.8/dev_guide_binary.html). The functionality is implemented on CPUs and Intel GPUs.
+
+## Intel Graphics Products
+* Introduced support for [GenIndex](https://oneapi-src.github.io/oneDNN/v3.8/dev_guide_op_genindex.html) operation in Graph API.
+
+## Intel Architecture Processors
+* Introduced support for `f32` convolution with `fp16` compressed weights.
+* Enabled support for int8 or int4 compressed weights in matmul primitive. This functionality is implemented on Intel CPUs.
+
+## Generic SYCL backend
+* Introduced support for RNN Vanilla primitive forward propagation.
+* Introduced support for inner product primitive backward propagation.
+* Introduced support for group normalization primitive.
+* Improved precision of inner product primitive with sum post-op for larger shapes.
+
+## AArch64-based Processors
+* Enabled FP16 for JIT reorder kernels.
+* Enabled matmul static quantization.
+
+## NVIDIA GPUs
+* Introduces Graph API support.
+
+# Usability
+ 
+* Added support for group normalization primitive with [ONEDNN_ENABLE_PRIMITIVE](https://uxlfoundation.github.io/oneDNN/dev_guide_build_options.html#onednn-enable-primitive) build option.
+* Enabled ROCm 6 support for AMD GPUs.
+* Improved CMake integration when installing oneDNN with the Nvidia backend enabled.
+
+## AArch64-based Processors
+ * Default the number of threads to max for acl_threadpool. Fixes a crash in Tensorflow.
+ * Fixed scratchpad being ignored for some GEMMs. Reduces memory and speeds up execution.
+ * Fixed a bug in FP32 reorders where ACL would return incorrect results.
+
+# Validation
+* Added benchdnn option [`--execution-mode`](https://github.com/uxlfoundation/oneDNN/blob/rls-v3.8/tests/benchdnn/doc/knobs_common.md#--execution-mode) to test oneDNN functionality with SYCL Graph record/execute mode.
```

Comment:
would combine: 
* `benchdnn`:
   * Added  `--execution-mode` to test oneDNN functionality with SYCL* Graph record/execute mode.
   * Extended `--cold-cache` with cold TLB mode support.
   * Added `--bia-dt` to control bias data types in matmul, inner product, convolution, and deconvolution.
   * Extended `--dt` syntax in Graph API driver for per-tensor data type specification.